sys/shell/commands/sc_gnrc_icmpv6_echo.c

```diff
@@ -371,7 +371,7 @@ static void _print_reply(_ping_data_t *data, gnrc_pktsnip_t *icmpv6,
             data->num_recv++;
             dupmsg += 7;
         }
-        if (gnrc_netif_highlander() || (if_pid == KERNEL_PID_UNDEF) ||
+        if ((gnrc_netif_numof() == 1) || (if_pid == KERNEL_PID_UNDEF) ||
```

Comment:
This use is also legitimate. It is only deciding if the interface is printet for a link-local address or not, so there is no harm in printing it, if `GNRC_NETIF_SINGLE` is defined.

Reply:
Already removed it :wink: